pkg/cmd/server/start/start_allinone.go

```diff
@@ -105,16 +113,16 @@ func NewCommandStartAllInOne(basename string, out io.Writer) (*cobra.Command, *A
 	BindListenArg(listenArg, flags, "")
 	BindImageFormatArgs(imageFormatArgs, flags, "")
 
-	startMaster, _ := NewCommandStartMaster(basename, out)
-	startNode, _ := NewCommandStartNode(basename, out)
-	startNodeNetwork, _ := NewCommandStartNetwork(basename, out)
-	startEtcdServer, _ := NewCommandStartEtcdServer(RecommendedStartEtcdServerName, basename, out)
+	startMaster, _ := NewCommandStartMaster(basename, delegatedOutput)
```

Comment:
@deads2k so I'm confused -- how are we drawing this line? It's fine to set `MasterOptions` output to be the delegated wrapper, but these shouldn't be delegated? Do we just never want the `Generating node credentials...` string to be anything but `glog`, thereby not needing the wrapper?